server/src/test/java/io/crate/metadata/doc/DocIndexMetadataTest.java

```diff
@@ -1538,32 +1538,6 @@ public void testSchemaWithGeneratedColumn() throws Exception {
         Asserts.assertThat(((GeneratedReference) week).referencedReferences()).satisfiesExactly(isReference("ts"));
     }
 
-    @Test
-    public void testCopyToWithoutMetaIndices() throws Exception {
-        // regression test... this mapping used to cause an NPE
-        XContentBuilder builder = JsonXContent.builder()
-            .startObject()
-            .startObject("properties")
-            .startObject("description")
-            .field("type", "string")
-            .field("position", 1)
-            .array("copy_to", "description_ft")
-            .endObject()
-            .startObject("description_ft")
-            .field("type", "string")
-            .field("position", 2)
-            .field("analyzer", "english")
-            .endObject()
-            .endObject()
-            .endObject();
-
-        IndexMetadata metadata = getIndexMetadata("test1", builder);
-        DocIndexMetadata md = newMeta(metadata, "test1");
-
-        assertThat(md.indices().size(), is(1));
-        assertThat(md.indices().keySet().iterator().next(), is(new ColumnIdent("description_ft")));
-    }
-
```

Comment:
Removed this test as now we NEED "indices" to figure out that we are dealing with FT column.

From the change entry of https://github.com/crate/crate/commit/e25a1f5dd12c85372f20bcd8f070e635bdc1e1c7#diff-504ec52f497c9e52b4c06dbfec9b1c29c2dc29f8c1e7befad24d9e4339c8d9d4

> Fixed a NPE that occurred if a table was created using the ES API with a
    mapping that uses ``copy_to`` but does not define ``_meta.indices``
    
This was a fix for 0.55.0 where ES API was still a thing, now it's not possible to define FT without having "indices" in metadata.